addOns/grpc/src/main/java/org/zaproxy/addon/grpc/internal/ProtoBufMessageDecoder.java

```diff
@@ -56,9 +56,7 @@ public void decode(byte[] inputEncodedData) {
                     break;
                 }
             } catch (IOException e) {
-                decodedToString.setLength(0);
-                decodedToList.clear();
-                LOGGER.debug("Error while decoding the message", e.getMessage());
+                LOGGER.debug("Error decoding the message", e.getMessage());
```

Comment:
This needs e.g. `: {}` in the log message, otherwise the second parameter will not do anything.